### PR TITLE
Added package formats

### PR DESCRIPTION
**Pull request by marmuc680**
Added package formats supported in partner center.

.MSIXVC and .XVC (including GameCore)

### CODE REVIEW

**Comment by PRMerger9** [comment]
@marmuc680 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: windows-apps-src/publish/upload-app-packages.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ ms.localizationpriority: medium
...
+The **Packages** page is where you upload all of the package files (.msix, .msixupload, .msixbundle, .appx, .appxupload, and/or .appxbundle .MSIXVC .XVC (including GameCore)) for the app that you're submitting. You can upload all your packages for the same app on this page, and when a customer downloads your app, the Store will automatically provide each customer with the package that works best for their device. After you upload your packages, you’ll see a table indicating [which packages will be offered to specific Windows 10 device families](#device-family-availability) (and earlier OS versions, if applicable) in ranked order.
```
Comment (hickeys) [inline windows-apps-src/publish/upload-app-packages.md]: Please move the "and/or" that is currently after ".appxupload" and put it after ".MSIXVC"

Please also add commas between each file extension in the list. 

Then it looks good to go!

**Review by hickeys — CHANGES_REQUESTED** [review]

**Comment by ktoliver** [comment]
Hi @marmuc680 - As discussed in email, this update should be made in the private repo. We'll close this PR. Thanks very much.

**Comment by ktoliver** [comment]
To edit an article in the private repo from within the documentation, after you select **Edit** on an article page,  add “-pr” to the repo name in the URL. 

image

Also, [SpineEdit](https://review.docs.microsoft.com/help/contribute/contribute-tool-spineedit?branch=master) is a Chrome extension that opens the edit option in the private repo, if there is one and if you have access to it.

<hr>

Related links in the [internal Microsoft Docs contributor guide](https://review.docs.microsoft.com/help/contribute/?branch=master):

[**Public vs. private repos**](https://review.docs.microsoft.com/help/contribute/contribute-authors-in-public-repo?branch=master) | [**Move a commit to a different repo (public -> private)**](https://review.docs.microsoft.com/help/contribute/contribute-how-to-manage-act-guidelines-public-pr?branch=master#pull-public-commits-to-the-private-repo-for-updates-staging-publication) | [**GitHub for content reviews**](https://review.docs.microsoft.com/help/contribute/contribute-how-to-write-pull-request-reviews?branch=master)
[**Quick reference for contributing to the docs**](https://review.docs.microsoft.com/help/contribute/contribute-quick-reference?branch=master) | [**Make changes to a Docs article**](https://review.docs.microsoft.com/help/contribute/edit-doc-with-vs-code-and-git-cli?branch=master) | [**PR review quality criteria**](https://review.docs.microsoft.com/help/contribute/contribute-how-to-write-pull-request-quality-criteria?branch=master)

**Comment by ktoliver** [comment]
Per Teams convo, we'll reopen this PR.

**Comment by ktoliver** [comment]
Conversation moved to email

**Pull request closed without merging**